resources/assets/js/features/public-profiles/actions.js

```diff
@@ -1,24 +1,27 @@
+import * as actions from './actionTypes';
 import { securedRequest } from 'app/services/RequestService';
-import moment from 'moment';
 
-const DriverProfileService = {
-    getDriverProfile(id) {
-        let response;
-        return response = {
+export const getProfileSuccess = profile => ({
```

Comment:
in this action you're going to set global state, this action doesn't search anything in existing state or server api, set appropriate name for it